circus-train-aws/src/test/java/com/hotels/bdp/circustrain/aws/AssumeRoleCredentialProviderTest.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.bdp.circustrain.aws;
+
+import org.apache.hadoop.conf.Configuration;
+import org.junit.Test;
+
+public class AssumeRoleCredentialProviderTest {
```

Comment:
Is it not possible to have a typical() test here ?

Reply:
no see the discussion me and @massdosage  had

Reply:
Read Adrian's earlier same comment and got my answer.